### DISCUSSION TITLE
Comments and discussions on "diff view"

### DISCUSSION THREAD

**Discussion by johntmyers** [GitbookIO/community#226] [Feature Requests]
### What's your feature/idea?

It would be nice to be able to provide comments when in "diff view" while reviewing a change request. Basically similar to how you'd review and discuss a PR on GitHub. When you can't see what was actually changed, it's hard to leave comments on something. Also even when you are not in diff view and you click on an existing comment, the page jumps to where the comment was left but it's still not very clear based on the minimal highlighting that is done.

### What's your desired outcome?

To be able to review change requests more similarly to how you would do a PR review in GitHub and actually have meaningful discussions about changes, not just content in general.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by dahveed-rb** [comment]
+1 on this one, we are trying to use gitbook in our product development process and without having this feature available, it makes it very difficult for managers to review what's changed in the feature plans and to leave feedback in a streamlined manner. Currently they have to jump back and forth and it seems counter intuitive, but maybe there is a reason why it's not a feature? 

There is also a use case where we might want to leave feedback asking why something was removed or suggesting to bring it back or something along those lines that would be beneficial to have, although not as important as being able to comment on additions.

**Comment by SamyPesse** [comment]
We just shipped this improvement. You can now comment on blocks when viewing diffs:


https://github.com/user-attachments/assets/1843317b-55de-48e8-baaf-ce6dce2bb682
